src/main/java/build/buildfarm/instance/shard/ShardInstance.java

```diff
@@ -1425,6 +1425,7 @@ protected void validatePlatform(
                       "property '%s' value was not a valid integer: %s",
                       property.getName(), property.getValue()));
         }
+      } else if (property.getName().equals("gpu")) {
```

Comment:
This seems extremely narrow - observe the provisioned queues to see what should be ignored here, rather than hardcoding gpu

Reply:
A little better now.  The validation is checked by the operationQueue so it can be consistent with how it decides eligibility